Lib/re/_constants.py

```diff
@@ -13,7 +13,7 @@
 
 # update when constants are added or removed
 
-MAGIC = 20220402
+MAGIC = 20220409
```

Comment:
No opcode is changed, is this necessary?

Reply:
The interface between the Python and the C code was changed. Updating MAGIC guaranteas that they are synchronized.